electron/utils.ts

```diff
@@ -126,17 +127,28 @@ const launchGame = async (appName: any) => {
     otherOptions,
     useGameMode,
     showFps,
-  } = settings[settingsName]
+    launcherArgs = '',
+    showMangohud,
+    audioFix,
+  } = settings[settingsName] as AppSettings
 
   let wine = `--wine ${wineVersion.bin}`
   let prefix = `--wine-prefix ${winePrefix}`
 
-  envVars = otherOptions
   const isProton = wineVersion.name.startsWith('Proton')
   prefix = isProton ? '' : `--wine-prefix ${winePrefix}`
 
+  const options = {
```

Comment:
Liked this approach since it will make it easier to add more options in the future.